rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -249,6 +252,115 @@ def testQuery_noSummaries(self):
+      with FakeClock(when):
+        participant = self.send_post('Participant', {"providerLink": [self.provider_link]})
+        participant_id = participant['participantId']
+        self.send_consent(participant_id)
+        # Populate some answers to the questionnaire
+        answers = {
+          'race': RACE_WHITE_CODE,
+          'genderIdentity': PMI_SKIP_CODE,
+          'firstName': self.fake.first_name(),
+          'middleName': self.fake.first_name(),
+          'lastName': self.fake.last_name(),
+          'zipCode': '78751',
+          'state': PMI_SKIP_CODE,
+          'streetAddress': '1234 Main Street',
+          'city': 'Austin',
+          'sex': PMI_SKIP_CODE,
+          'sexualOrientation': PMI_SKIP_CODE,
+          'phoneNumber': '512-555-5555',
+          'recontactMethod': PMI_SKIP_CODE,
+          'language': PMI_SKIP_CODE,
+          'education': PMI_SKIP_CODE,
+          'income': PMI_SKIP_CODE,
+          'dateOfBirth': datetime.date(1978, 10, 9),
+          'CABoRSignature': 'signature.pdf',
+        }
+      self.post_demographics_questionnaire(participant_id, questionnaire_id, time=when, **answers)
+      return participant
+
+    # Create the first batch and fetch their summaries
+    first_batch = [setup_participant(t1) for _ in range(5)]
+    first_batch.extend([setup_participant(t2) for _ in range(2)])
+    first_batch.extend([setup_participant(t3) for _ in range(3)])
+    url = 'ParticipantSummary?_sort=lastModified&_sync=true&awardee=PITT'
+    response = self.send_get(url)
+    # We have the same number of participants as summaries
+    self.assertEqual(len(response['entry']), len(first_batch))
+    last_modified_list = list()
+    first_batch_list = list()
+    for i in response['entry']:
+      last_modified_list.append(i['resource']['lastModified'])
+    for i in first_batch:
+      first_batch_list.append(i['lastModified'])
+
+    self.assertListEqual(last_modified_list, sorted(first_batch_list))
+    # With the same ID's (they're the same participants)
+    self.assertEqual(
+      sorted([p['participantId'] for p in first_batch]),
+      sorted([p['resource']['participantId'] for p in response['entry']]),
+    )
+    self.assertEqual(sorted([p['resource']['lastModified'] for p in response['entry']]),
+                      [p['resource']['lastModified'] for p in response['entry']])
+
+    # Get the next chunk with the sync url
+    # Verify that this is, in fact, a sync URL - not a next
+    sync_url = response['link'][0]['url']
+    index = sync_url.find('ParticipantSummary')
+    self.assertEqual(response['link'][0]['relation'], 'sync')
+
+    # Verify that the next chunk has no results (everyone is accounted for)
+    response2 = self.send_get(sync_url[index:])
+    self.assertEqual(len(response2['entry']), 0)
+    self.assertEqual(response2['entry'], [])
+
+    # Create a second batch
+    second_batch = [setup_participant(t4) for _ in range(10)]
+    response3 = self.send_get(sync_url[index:])
```

Comment:
See above, seems like you should be getting back some of the same participants as before?

Reply:
done.